src/gui/reports/ReportsWidgetHibp.cpp

```diff
@@ -198,6 +235,7 @@ void ReportsWidgetHibp::makeHibpTable()
     }
 
     m_ui->hibpTableView->resizeRowsToContents();
+    m_ui->hibpTableView->sortByColumn(2, Qt::DescendingOrder);
```

Comment:
`setSortingEnabled(true);` of line 91 automatically calls `sortByColumn()` and on line 90 we already do `setSortIndicator(2, Qt::DescendingOrder)`
```suggestion
```